sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/security/validation/SdaContainerCustomizer.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
```

Comment:
Is the header provided by legal? I thought adding the date is not needed by German law.

Reply:
Will create a subtask to clarify the license header. Just saw that we're also not using a header in sda-dropwizard-commons.